examples/example_user_input_material.py

```diff
@@ -56,8 +56,10 @@
     or 'bm2' (birch-murnaghan 2nd order, if you choose to ignore temperature
        (your choice in geotherm will not matter in this case))
     or 'bm3' (birch-murnaghan 3rd order, if you choose to ignore temperature
-        (your choice in geotherm will not matter in this case))"""
-    method = 'slb3'
+        (your choice in geotherm will not matter in this case)
+    or 'vinet' (vinet equation of state, if you choose to ignore temperature
+        (your choice in geotherm will not matter in this case)))"""
+    method = 'vinet'
```

Comment:
Is there a reason you switch to "vinet" here? I assume slb3 is a better default?